adala/runtimes/_litellm.py

```diff
@@ -232,6 +217,33 @@ class LiteLLMChatRuntime(InstructorClientMixin, Runtime):
 
     model_config = ConfigDict(extra="allow")
 
+    @property
+    def canonical_model_provider_string(self):
+        """provider_name/model_name"""
+        try:
+            return match_model_provider_string(self.model)
+        except NoModelsFoundError:
+            logger.info(
+                f"Model {self.model} not found in litellm model map for provider {self.provider}. This is likely a single-model deployment."
+            )
+        except Exception as e:
+            logger.exception(
```

Comment:
probably logger.error right..?

Reply:
logger.exception provides the full stack trace, and is otherwise the same. It's helpful for debugging

Reply:
if we get here then this is unexpected and we have another edge case to fix, in contrast to NoModeslFoundError which is expected